flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -204,6 +204,12 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
                 #endif
             }
 
+            if let sdk = arguments["sdk"] as? [String: Any?],
+               let sdkName = sdk["name"] as? String,
+               let sdkVersion = sdk["version"] as? String {
+                PrivateSentrySDKOnly.setSdkName(sdkName, andVersionString: sdkVersion)
```

Comment:
Check the name and version nullability similarly to how its done on Android.

Reply:
Not sure i understand, we are unpacking the optionals with `if let`, or do you mean something else? It should be similar to how the Kotlin plugin file does it.

<img width="551" alt="Bildschirm­foto 2023-01-16 um 10 13 08" src="https://user-images.githubusercontent.com/3984453/212640772-f4d87d39-a457-4940-b21f-0c54c8bcc2e3.png">

Reply:
https://github.com/getsentry/sentry-dart/pull/1222/files#r1071003637
This is needed